code/game/gamemodes/malfunction/newmalf_ability_trees/HARDWARE.dm

```diff
@@ -114,35 +114,41 @@
 	set_security_level("delta")
 
 	if(timer > stage1)
-		radio.autosay("Warning: Brute force attempt on primary firewall detected.", "Self-Destruct Control")
+		radio.autosay("Critical: Brute force attempt on primary firewall detected.", "Station Authentication Control")
 	else if(timer > stage2)
-		radio.autosay("Warning: Brute force attempt on backup firewall detected.")
+		radio.autosay("Alert: Brute force attempt on backup firewall detected.", "Station Authentication Control")
 	else
-		radio.autosay("Self destruct sequence has been activated. Self-destructing in [timer] seconds.", "Self-Destruct Control")
+		radio.autosay("Emergency: Self destruct sequence has been activated. Self-destructing in [timer] seconds.", "Station Authentication Control")
 
 	
 	while(timer)
 		sleep(10)
 		var/obj/machinery/nuclearbomb/station/N = nuke.resolve()
 		if(!user || !user.bombing_station || user.stat == DEAD || !N)
-			radio.autosay("Self destruct sequence has been cancelled.", "Self-Destruct Control")
+			if(timer < stage2)
+				radio.autosay("Self destruct sequence has been cancelled.", "Station Authentication Control")
+			else
+				radio.autosay("Brute force attempt has ceased.", "Station Authentication Control")
 			return
 		if(N.auth)
-			radio.autosay("Local Override Engaged - Self-Destruct canceled.", "Self-Destruct Control")
+			if(timer < stage2)
+				radio.autosay("Local Override Engaged - Self-Destruct canceled.", "Station Authentication Control")
+			else
+				radio.autosay("Local Override Engaged - Network Connection Disabled.", "Station Authentication Control")
 			user.bombing_station = 0
 			return
 		if(timer == stage1+1)
-			radio.autosay("Critical: Primary firewall bypassed.")
-			radio.autosay("Warning: Brute force attempt on backup firewall detected.")
+			radio.autosay("Alert: Primary firewall bypassed.")
+			radio.autosay("Alert: Brute force attempt on backup firewall detected.")
 			user.bombing_time = stage1 //Further attempts will only take 900 seconds
 		if(timer == stage2+1)
-			radio.autosay("Critical: Backup firewall failed.")
-			radio.autosay("Self destruct sequence has been activated. Self-destructing in [timer] seconds.", "Self-Destruct Control")
+			radio.autosay("Emergency: Backup firewall failed.")
+			radio.autosay("Self destruct sequence has been activated. Self-destructing in [timer] seconds.", "Station Authentication Control")
 			user.bombing_time = stage2 //Further attempts will only take 600 seconds
 		if(timer in list(2, 3, 4, 5, 10, 30, 60, 90, 120, 240, 300)) // Announcement times. "1" is not intentionally included!
-			radio.autosay("Self destruct in [timer] seconds.", "Self-Destruct Control")
+			radio.autosay("Self destruct in [timer] seconds.", "Station Authentication Control")
 		if(timer == 1)
-			radio.autosay("Self destructing now. Have a nice day.", "Self-Destruct Control")
+			radio.autosay("Self destructing now. Have a nice day.", "Station Authentication Control")
```

Comment:
Hyphenate `Self-destructing`. Although, perhaps it would sound better as, `Self-destruct sequence initiated. Have a nice day.`